.github/workflows/ci-unit-tests-go-tip.yml

```diff
@@ -22,11 +24,16 @@ jobs:
     - name: Install Go Tip
       uses: ./.github/actions/setup-go-tip
 
-    - name: Install tools
-      run: make install-test-tools
-
```

Comment:
?

Reply:
Oh, it's because this step was included in the test-ci make target.